tracks/pep8/lab1-1/config.ini

```diff
@@ -1 +1,3 @@
 title=Addition simple
+time_goal=24
```

Comment:
just a better readability for the end user and some parallelism with the mental JSON object one should have about a mission. Don't you represent everything has JSON?